src/Data/Quaternion/Rotation.purs

```diff
@@ -107,3 +112,24 @@ act (Rotation p) v =
   unsafePartial $ case Vec.toArray v of
     [x, y, z] ->
       vectorPart (Quaternion zero x y z `conjugateBy` p)
+
+normalize :: Rotation Number -> Rotation Number
```

Comment:
Sure, I'll commit that for your review. Does that warrant a new PR, or would you end up pulling from the new state of my master when merging this one?